BentoTests/ConcatenationTests.swift

```diff
@@ -70,7 +70,7 @@ class ConcatenationTests: XCTestCase {
         let optional: String? = "something"
 
         let result: Section<TestSectionId, TestRowId> = section
-            |---? .some(optional) { _ in
+            |---? optional.map { _ in
                 node
             }
 
```

Comment:
Do we have tests for `.iff` ?

If not can you please add one in this file?

Reply:
`|-? .iff` tests go to `IfTests.swift`, which are unchanged in this PR.